make/Main.gmk

```diff
@@ -1259,7 +1278,7 @@ all-images: product-images test-image all-docs-images
 # all-bundles packages all our deliverables as tar.gz bundles.
 all-bundles: product-bundles test-bundles docs-bundles static-libs-bundles
 
-ALL_TARGETS += buildtools hotspot hotspot-libs hotspot-gensrc gensrc gendata \
+ALL_TARGETS += buildtools hotspot hotspot-libs hotspot-static-libs hotspot-gensrc gensrc gendata \
```

Comment:
Can you add a newline to try to keep line length in check here. No need to reformat the whole block, just don't let random lines shoot way over 80.

Reply:
Done, thanks.